app/feature/feature-choose-tier/src/main/kotlin/com/hedvig/android/feature/change/tier/ui/stepcustomize/SelectTierDestination.kt

```diff
@@ -319,19 +333,31 @@ private fun CustomizationCard(
           items = tierSimpleItems,
         ),
         size = Small,
-        hintText = stringResource(R.string.TIER_FLOW_COVERAGE_PLACEHOLDER),
+        hintText = hintText,
         chosenItemIndex = chosenTierIndex,
         onDoAlongWithDismissRequest = onSetTierBackToPreviouslyChosen,
+        modifier = Modifier.accessibilityForDropdown(
+          labelText = stringResource(R.string.TIER_FLOW_COVERAGE_LABEL),
+          selectedValue = chosenTierVoiceDescription,
+          isEnabled = isTierChoiceEnabled,
+        ),
       ) { onDismissRequest ->
         val listOfOptions = tiers.map { pair ->
+          val tierDescription = pair.first.tierDescription ?: ""
+          val price = stringResource(R.string.TALKBACK_PRICE)
+          val premiumDescription = pair.second.getPerMonthDescription()
+          val voiceDescription = "${pair.first.tierDisplayName}, $tierDescription, $price: ${stringResource(
+            R.string.TALKBACK_FROM,
+          )} $premiumDescription"
           ExpandedRadioOptionData(
             chosenState = if (chosenTierInDialog == pair.first) Chosen else NotChosen,
             title = pair.first.tierDisplayName ?: "-",
-            premium = stringResource(R.string.TIER_FLOW_PRICE_LABEL, pair.second.amount.toInt()),
+            premiumString = stringResource(R.string.TIER_FLOW_PRICE_LABEL, pair.second.amount.toInt()),
             tierDescription = pair.first.tierDescription,
             onRadioOptionClick = {
               onChooseTierInDialogClick(pair.first)
             },
+            voiceoverDescription = voiceDescription,
```

Comment:
In all the places we say "voiceover" perhaps we can say "contentDescription" instead to keep it the same as what is used by compose anyway?